cmd/user.go

```diff
@@ -0,0 +1,125 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/base64"
+	"github.com/cloudflare/cfssl/log"
+	"github.com/k0sproject/k0s/pkg/certificate"
+	"github.com/k0sproject/k0s/pkg/constant"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+func init() {
+	userCreateCmd.Flags().StringVar(&groups, "groups", "", "Specify groups")
+
+	userCmd.AddCommand(userCreateCmd)
+}
+
+var (
+	groups string
+
+	userKubeconfigTemplate = template.Must(template.New("kubeconfig").Parse(`
+apiVersion: v1
+clusters:
+- cluster:
+    server: {{.JoinURL}}
+    certificate-authority-data: {{.CACert}}
+  name: k0s
+contexts:
+- context:
+    cluster: k0s
+    user: {{.User}}
+  name: k0s
+current-context: k0s
+kind: Config
+preferences: {}
+users:
+- name: {{.User}}
+  user:
+	client-certificate-data: {{.ClientCert}}
+	client-key-data: {{.ClientKey}}
```

Comment:
When testing this template, kubectl fails, because the indentation is used generated using tabs instead of spaces:
```
error: error loading config file "/home/ubuntu/.kube/config": yaml: line 19: found character that cannot start any token
```
Please verify that the resulting Yaml file does not include tabs, since YAML forbids it.